### DISCUSSION TITLE
멀티플레이 설정 동일 PC에서 확인 방법

### DISCUSSION THREAD

**Discussion by runtostar** [naverz/zepeto-studio-kor#66] [5. Q&A]
동일 PC에서 서버/클라이언트 동시 실행 가능한가요?
그리고 구동 중인 서버의 IP 주소는 '서버 로그 창'에서 확인할 수 있다고 하는데, '서버 로그 창'은 어디서 볼 수 있을까요?
![image](https://user-images.githubusercontent.com/95363110/145702724-35b88890-3603-4191-b6b8-d2169267d544.png)

**Comment by x4cf** [comment]
동시 실행을 해도 별도의 설정 없이는 정상적으로 접속되지는 않을것으로 예상됩니다.

윈도우에 보시면 팝업창을 노출시킬 수 있습니다.
<img width="973" alt="스크린샷 2021-12-12 오후 6 26 47" src="https://user-images.githubusercontent.com/90679926/145707147-5327432d-e7d0-4631-8f1e-5f5c3478a0c4.png">

**Reply by runtostar** [reply]
답변 감사드립니다. 모바일과 PC로 각각 접속하니 둘 다 접속은 되긴 하는데, 서로의 모습이 보이질 않네요. local player만 확인되고 remote player는 안 보이는 현상이 있는데 혹시 이런 문제 있으셨는지요?

![image](https://user-images.githubusercontent.com/95363110/145748184-388663b8-2385-4c1b-b6db-de27441e88d5.png)
